app/controllers/application_controller.rb

```diff
@@ -1,5 +1,12 @@
 class ApplicationController < ActionController::Base
-  def current_user
-    User.first
+  protect_from_forgery with: :exception
+
```

Comment:
- **[OPTIONAL]** It would be nice to ensure only the logged in users can use the app. You can use the `authenticate_user!` method.

  ```
  before_action :authenticate_user!
  ```